site/pages/docs/versions/v4.0/v4.0.21-en.hbs

```diff
@@ -0,0 +1,102 @@
+---
+{
+	"title": "v4.0.21 release notes",
+	"language": "en",
+	"description": "v4.0.21 release notes - Version history - Web Experience Toolkit (WET)",
+	"altLangPrefix": "v4.0.21,
```

Comment:
Didn’t saw that!!
DAMN!!! C poche ca!

J’ai pousser mes changements ☺


Merci/Thanks,

Maxim Martel

From: Laurent Goderre [mailto:notifications@github.com]
Sent: April-11-16 3:16 PM
To: wet-boew/wet-boew
Cc: Martel, Maxim M [NC]
Subject: Re: [wet-boew/wet-boew] WET release notes for v4.0.21 (#7493)


In site/pages/docs/versions/v4.0/v4.0.21-en.hbs<https://github.com/wet-boew/wet-boew/pull/7493#discussion_r59263653>:

> @@ -0,0 +1,102 @@

> +---

> +{

> +     "title": "v4.0.21 release notes",

> +     "language": "en",

> +     "description": "v4.0.21 release notes - Version history - Web Experience Toolkit (WET)",

> +     "altLangPrefix": "v4.0.21,

Missing double quote at the end here

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/wet-boew/wet-boew/pull/7493/files/5d6ed2e6f215960fe3d1f58e2cfbde95a03c012d#r59263653>